data/tools/wmlindent

```diff
@@ -299,7 +299,16 @@ def convertor(linefilter, arglist, exclude):
                 print("wmlindent: no WML file found, exiting", file=sys.stderr)
                 sys.exit(1)
 
+def sigint_handler(signal, frame):
+    """This function defines what happens when the SIGINT signal is encountered by pressing ctrl-c during runtime. 
+    When ctrl-c is pressed, a one-line message is displayed and Python exits with Status 0, which refers to successful termination.
+    This overrides Python's default behavior of displaying a traceback when ctrl-c is pressed.
+    """
+    print ('Aborted by pressing ctrl-c')
```

Comment:
Is this os independent? Normally, I have to do cmd+c for the same thing on macos.

Reply:
It's the same message on all OS as of latest commit in the PR. Would it suffice if i change the message to say 
>Aborted using Keyboard Interrupt

?

Reply:
> Is this os independent? Normally, I have to do cmd+c for the same thing on macos.

Cmd-C is only the equivalent of ctrl-c when other OSes are using ctrl-c to mean "copy"; it's still ctrl-c when it's used to mean "kill this process running in a terminal" though

Reply:
> Is this os independent? Normally, I have to do cmd+c for the same thing on macos.

???

Ctrl+C for interrupt is still Ctrl+C on Mac (although Cmd+period also works).

Reply:
Still a bit odd to reference ctrl-c specifically since it is just one way to send an INT signal.

Reply:
What's the other way to send INT?

Reply:
You use kill to send signals. In general the syscall or on the command line there is a kill command and your shell probably has it as a builtin as well.

Reply:
If you're talking about `kill` command that could even be sent without using keyboard interrupt. In that case does "Runtime Interrupted by User" sound better?